action.yml

```diff
@@ -217,6 +217,12 @@ inputs:
     description:
       'The path to mount the workspace inside the docker container. For windows, leave out the drive letter. For example
       c:/github/workspace should be defined as /github/workspace'
+  errorWhenMissingUnityBuildResults:
+    default: true
+    required: false
+    description:
+      'Check if Unity build product is present after build, and error if not. Set to false to not check, useful if
+      producing alternative build products.'
```

Comment:
This is effectively a discrepancy in the application (builder and/or container). I feel adding an option in the (functional) public api isn't the answer to this technical issue. 

We should probably agree on how the application should handle this case and not expose anything.